docs/writing-docs/tutorials/basics.md

```diff
@@ -168,6 +168,22 @@ forever(function() {
 ```
 ````
 
+## Accordion/hidden hints
+If you want to provide extra information without having to divert the coder's attention, you can include content in an "accordion" style hint control. 
```

Comment:
It would maybe be worth dropping a link to https://makecode.com/writing-docs/tutorials/control-options#explicit-hints here? Or ideally images / snippets that are inside ~hints wouldn't trigger the 'hide in hint bubble' logic ever, but that may be a bit more complicated to implement.

Reply:
Great point, Joey ^